packages/docs/content/components/button.mdx

```diff
@@ -11,172 +11,111 @@ medicare:
   sketchLink: Vr4yyWw
 ---
 
-import { Alert, Button, ArrowIcon, TextInput } from '@cmsgov/design-system';
+import { ArrowIcon, Button } from '@cmsgov/design-system';
 
-## Default Button
+## Examples
 
-<EmbeddedExample>
-  <div>
-    <Button className="ds-u-margin-right--1">Default</Button>
-    <Button className="ds-c-button--hover ds-u-margin-right--1">Hover</Button>
-    <Button className="ds-c-button--active ds-u-margin-right--1">Active</Button>
-    <Button className="ds-c-button--focus ds-u-margin-right--1">Focus</Button>
-    <Button disabled>Disabled</Button>
-  </div>
-</EmbeddedExample>
+The following tables give an overview of the different permutations of buttons available in the selected design system. The controlling traits are the named _variation_, whether it's the _main_ vs. _alternate_ color, and the _on-light_ vs. _on-dark_ context. Each of these traits has its own example later on showing how to apply it.
 
-## Primary Button
+<ButtonVariationsTable />
 
-<EmbeddedExample>
-  <div>
-    <Button className="ds-u-margin-right--1" variation="primary">
-      Default
-    </Button>
-    <Button className="ds-c-button--hover ds-u-margin-right--1" variation="primary">
-      Hover
-    </Button>
-    <Button className="ds-c-button--active ds-u-margin-right--1" variation="primary">
-      Active
-    </Button>
-    <Button className="ds-c-button--focus ds-u-margin-right--1" variation="primary">
-      Focus
-    </Button>
-    <Button disabled variation="primary">
-      Disabled
-    </Button>
-  </div>
-</EmbeddedExample>
+### Outline buttons
 
-## Success Button
+<StorybookExample
+  componentName="button"
+  sourceFilePath="components/Button/Button.tsx"
+  storyId="components-button--default&args=children:Outline"
+/>
 
-<EmbeddedExample>
-  <div>
-    <Button className="ds-u-margin-right--1" variation="success">
-      Default
-    </Button>
-    <Button className="ds-c-button--hover ds-u-margin-right--1" variation="success">
-      Hover
-    </Button>
-    <Button className="ds-c-button--active ds-u-margin-right--1" variation="success">
-      Active
-    </Button>
-    <Button className="ds-c-button--focus ds-u-margin-right--1" variation="success">
-      Focus
-    </Button>
-    <Button disabled variation="success">
-      Disabled
-    </Button>
-  </div>
-</EmbeddedExample>
+### Solid buttons
 
-## Transparent Button
+<StorybookExample
+  componentName="button"
```

Comment:
Please update the `componentName` prop to be unique for each usage of `<StorybookExample>` because it is used to generate a unique ID and title for each iframe.

Reply:
Good to know, thanks!